perception/autoware_occupancy_grid_map_outlier_filter/src/occupancy_grid_map_outlier_filter_node.cpp

```diff
@@ -106,13 +106,13 @@ namespace autoware::occupancy_grid_map_outlier_filter
 {
 RadiusSearch2dFilter::RadiusSearch2dFilter(rclcpp::Node & node)
 {
-  search_radius_ = node.declare_parameter("radius_search_2d_filter.search_radius", 1.0f);
+  search_radius_ = node.declare_parameter<float>("radius_search_2d_filter.search_radius");
   min_points_and_distance_ratio_ =
-    node.declare_parameter("radius_search_2d_filter.min_points_and_distance_ratio", 400.0f);
-  min_points_ = node.declare_parameter("radius_search_2d_filter.min_points", 4);
-  max_points_ = node.declare_parameter("radius_search_2d_filter.max_points", 70);
+    node.declare_parameter<float>("radius_search_2d_filter.min_points_and_distance_ratio");
+  min_points_ = node.declare_parameter<int>("radius_search_2d_filter.min_points");
+  max_points_ = node.declare_parameter<int>("radius_search_2d_filter.max_points", 70);
   max_filter_points_nb_ =
-    node.declare_parameter("radius_search_2d_filter.max_filter_points_nb", 15000);
+    node.declare_parameter<int>("radius_search_2d_filter.max_filter_points_nb", 15000);
```

Comment:
@oguzkaganozt 
Thank you for your PR.
Why is the default parameter written in both code and config file?

Reply:
Resolved, please review again